src/renderer/Components/CreatingPackage/PackagePreview/index.jsx

```diff
@@ -1,8 +1,54 @@
-import Order from "./Order";
+import axios from "axios";
+import { S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
 import usePackageStore from "@renderer/store";
+import Order from "./Order";
 
 function PackagePreview() {
   const { orders } = usePackageStore();
+  const getOrders = usePackageStore((state) => state.getOrders);
```

Comment:
```suggestion
  const { getOrders } = usePackageStore();
```
Store 에서 getOrders 함수를 가져오는경우는 이렇게 적어주시면 된답니다! 😚 훅처럼 사용 할 수 있어서 참 편하지요?